sklearn/preprocessing/data.py

```diff
@@ -231,9 +235,12 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
         Per feature relative scaling of the data.
     """
 
-    def __init__(self, feature_range=(0, 1), copy=True):
+    def __init__(self, feature_range=(0, 1), copy=True,
+                 continuous_features=None):
         self.feature_range = feature_range
         self.copy = copy
+        self.continuous_features_ = continuous_features
```

Comment:
Attributes with a trailing underscore should, by our conventions, be learnt from data at `fit`. In this case, there is no need for such an attribute. You can instead save the value of `continuous_features` into `self.continuous_features` and use this instead of `self.continuous_features_`.